docs/development.md

```diff
@@ -200,11 +217,14 @@ echo "Machine SSH key generated in ${SSH_KEY_FILE}"
 export AZURE_SSH_PUBLIC_KEY=$(cat "${SSH_KEY_FILE}.pub" | base64 | tr -d '\r\n')
 ```
 
-#### Creating a test cluster
+##### Creating the cluster
+
+⚠️ Make sure you followed the previous two steps to build the dev image and set the required environment variables before proceding.
 
-Ensure kind has been reset:
+Ensure dev environment has been reset:
```

Comment:
Does kind need to be reset every deployment?  With the `kind load` being run would it be possible to leave the kind cluster running and update the images/manifests?

Reply:
Good question. I think it's better to reset every time because if the kind cluster already exists it can't be created and we don't expect kind clusters to be long lived so I think it's better to create a new cluster from scratch every time and also install the latest version of the controller manager and CRDs since this is mostly for dev use (and testing new code).